hplsql/src/main/java/org/apache/hive/hplsql/Exec.java

```diff
@@ -328,16 +334,16 @@ public String callStackPop() {
    */
   public Var findVariable(String name) {
     Var var;
-    String name1 = name;
+    String name1 = name.toUpperCase();
     String name1a = null;
     String name2 = null;
     Scope cur = exec.currentScope;
     Package pack;
     Package packCallContext = exec.getPackageCallContext();
     ArrayList<String> qualified = exec.meta.splitIdentifier(name);
     if (qualified != null) {
-      name1 = qualified.get(0);
-      name2 = qualified.get(1);
+      name1 = qualified.get(0).toUpperCase();
```

Comment:
splitIdentifier() is also used with table names and column names, I think it's better to leave column and table names alone as it would modify the HiveQL queries and it's rather Hive's territory than HPLSQL.